test/cql-pytest/conftest.py

```diff
@@ -234,11 +234,6 @@ def has_tablets(cql):
     with new_test_keyspace(cql, " WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor': 1}") as keyspace:
         return keyspace_has_tablets(cql, keyspace)
 
-@pytest.fixture(scope="function")
-def xfail_tablets(request, has_tablets):
-    if has_tablets:
-        request.node.add_marker(pytest.mark.xfail(reason='Test expected to fail with tablets experimental feature on'))
-
```

Comment:
I decided not to remove this function from my Alternator test PR, because even if I do not use it, it wasn't trivial to get this function working correctly, and unlike Avi I'm not convinced we'll never ever want to use it. But if you want to remove it, you can do it and we can always find it in the history if we need it again.

Reply:
I don't think there is any point in keeping this around, with no user in sight.